crm_claim_rma/models/crm_claim.py

```diff
@@ -204,6 +214,12 @@ def __create_lines_from_invoice_packages(self):
                     self,
                 ))
                 lines.append((0, 0, line))
+            if not line:
```

Comment:
IMO, would be better to test `if moves.quant_ids` before the loop at line 199, and change this condition to `else`.